site/src/atomics/aspect-ratio.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Aspect ratio
+description: Aspect ratio related atomic css classes for the Atlas Design System
+template: standard
+classType: Atomics
+classPrefixes:
+  - aspect-ratio
+---
+
+# Aspect Ratio Atomics
+
+Aspect ratio atomics are used to set the width-to-height ratio of an element, even if the parent container or viewport size changes.
```

Comment:
Suggestion only: perhaps write up your general views on using width with this? Something like: 

> ... Note that in some scenarios, elements will need to have `width: 100%` and/or `height: 100%` set. This can be done with the use of atomics such as `width-full` or a component like [`image](./components.

I don't believe we have any height atomics at the moment, so depending on your thoughts, we can commit that part of the above statement.